pkg/scuzz/ata.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package scuzz
+
+const (
+	oldSchoolBlockLen = 512
+)
+
+type (
+	// Cmd is an ATA command
+	Cmd uint8
+	// Block is a classc ATA block used by hdparm etc.
```

Comment:
Don't mention hdparm or whatever that is. Assume I have no idea what hdparm is and I don't have time to read its docs.

// Block is a classic ATA block.
//
// It is not completely clear ...

Reply:
done